kafka/metrics_test.go

```diff
@@ -386,11 +455,24 @@ func TestConsumerMetrics(t *testing.T) {
 		},
 	}
 
+	ignore := []string{
```

Comment:
Why do we need to ignore them? should we be able to check these metrics too as it's now reported by our own hook?